cloudformation/invoice-storage.yaml

```diff
@@ -0,0 +1,9 @@
+InvoiceStorageQueue:
+  Type: AWS::SQS::Queue
+  Properties:
+    KmsMasterKeyId: !GetAtt KmsKey.Arn
+
+InvoiceStorageTopic:
```

Comment:
I know this is still in draft but I thought I'd share something I came across which might impact this. 
Step 3 from the below says to prepend the SNS topic name with AmazonTextract.
https://docs.aws.amazon.com/textract/latest/dg/api-async-roles.html#api-async-roles-all-topics.

Reply:
I need to refer to the SNS topic in my extraction function so I've gone with AmazonTextractSNSTopic. Is that ok?

Reply:
Thanks Gareth, your reasoning makes sense. AmazonTextractRawInvoiceDataTopic it is!